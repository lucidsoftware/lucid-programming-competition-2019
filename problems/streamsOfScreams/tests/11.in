1000
ghost
notscary
notscary
notscary
ghost
notscary
ghost
ghost
notscary
ghost
boo!
ghost
notscary
notscary
notscary
ghost
boo!
notscary
notscary
boo!
ghost
notscary
boo!
boo!
ghost
notscary
ghost
notscary
notscary
ghost
boo!
ghost
notscary
boo!
boo!
ghost
notscary
notscary
ghost
ghost
boo!
notscary
ghost
notscary
boo!
boo!
boo!
boo!
notscary
boo!
ghost
boo!
ghost
ghost
boo!
notscary
boo!
notscary
boo!
boo!
ghost
ghost
ghost
boo!
ghost
boo!
notscary
notscary
notscary
ghost
notscary
ghost
boo!
notscary
ghost
ghost
notscary
ghost
notscary
boo!
boo!
notscary
boo!
ghost
ghost
ghost
notscary
ghost
ghost
ghost
notscary
ghost
ghost
notscary
notscary
boo!
notscary
notscary
ghost
ghost
boo!
notscary
boo!
boo!
notscary
notscary
boo!
boo!
ghost
boo!
notscary
boo!
ghost
ghost
notscary
notscary
boo!
ghost
notscary
ghost
notscary
ghost
boo!
ghost
ghost
boo!
boo!
notscary
boo!
ghost
notscary
boo!
boo!
ghost
ghost
notscary
ghost
ghost
boo!
ghost
ghost
ghost
notscary
ghost
notscary
ghost
ghost
notscary
ghost
notscary
ghost
notscary
boo!
ghost
ghost
boo!
ghost
ghost
notscary
ghost
notscary
notscary
boo!
ghost
notscary
notscary
ghost
ghost
notscary
boo!
boo!
boo!
boo!
notscary
boo!
ghost
boo!
ghost
ghost
notscary
boo!
notscary
notscary
notscary
ghost
boo!
ghost
notscary
notscary
ghost
notscary
notscary
boo!
notscary
boo!
notscary
ghost
notscary
notscary
notscary
boo!
ghost
boo!
ghost
notscary
ghost
notscary
boo!
ghost
ghost
ghost
notscary
boo!
ghost
boo!
boo!
boo!
ghost
boo!
ghost
ghost
boo!
ghost
notscary
notscary
ghost
ghost
notscary
boo!
notscary
notscary
ghost
ghost
boo!
notscary
notscary
ghost
boo!
boo!
ghost
boo!
notscary
ghost
ghost
notscary
ghost
boo!
notscary
boo!
notscary
notscary
notscary
boo!
boo!
boo!
boo!
notscary
boo!
ghost
ghost
notscary
notscary
boo!
boo!
boo!
boo!
notscary
notscary
notscary
boo!
notscary
boo!
boo!
boo!
notscary
notscary
ghost
ghost
ghost
boo!
boo!
ghost
notscary
boo!
boo!
notscary
notscary
notscary
notscary
ghost
ghost
notscary
boo!
ghost
notscary
notscary
ghost
ghost
ghost
boo!
boo!
ghost
notscary
notscary
ghost
notscary
ghost
boo!
notscary
ghost
boo!
notscary
notscary
notscary
notscary
boo!
ghost
boo!
boo!
notscary
notscary
notscary
ghost
ghost
ghost
boo!
boo!
ghost
ghost
ghost
notscary
ghost
boo!
boo!
ghost
notscary
notscary
notscary
ghost
notscary
notscary
notscary
ghost
boo!
notscary
ghost
boo!
notscary
notscary
notscary
boo!
notscary
boo!
notscary
boo!
notscary
notscary
notscary
notscary
boo!
boo!
notscary
notscary
notscary
ghost
notscary
ghost
boo!
notscary
boo!
ghost
ghost
notscary
ghost
notscary
notscary
boo!
notscary
notscary
ghost
notscary
boo!
notscary
boo!
ghost
notscary
boo!
boo!
notscary
ghost
notscary
notscary
ghost
boo!
boo!
boo!
ghost
boo!
ghost
boo!
notscary
ghost
ghost
ghost
notscary
ghost
boo!
ghost
notscary
boo!
notscary
notscary
boo!
boo!
boo!
notscary
notscary
notscary
ghost
notscary
ghost
boo!
notscary
ghost
boo!
notscary
notscary
ghost
boo!
boo!
notscary
notscary
ghost
ghost
boo!
ghost
notscary
ghost
notscary
boo!
boo!
boo!
boo!
ghost
notscary
boo!
ghost
notscary
notscary
boo!
boo!
ghost
notscary
ghost
notscary
notscary
notscary
boo!
notscary
ghost
notscary
boo!
boo!
boo!
notscary
boo!
ghost
notscary
notscary
notscary
ghost
notscary
notscary
notscary
boo!
ghost
ghost
notscary
boo!
boo!
notscary
notscary
boo!
ghost
notscary
notscary
notscary
notscary
boo!
notscary
boo!
ghost
notscary
notscary
boo!
boo!
ghost
ghost
notscary
notscary
boo!
boo!
ghost
boo!
notscary
notscary
notscary
ghost
notscary
notscary
boo!
boo!
boo!
notscary
notscary
boo!
notscary
ghost
boo!
ghost
boo!
notscary
boo!
boo!
ghost
boo!
notscary
ghost
notscary
notscary
notscary
ghost
ghost
boo!
ghost
ghost
ghost
ghost
boo!
notscary
ghost
notscary
ghost
notscary
notscary
ghost
notscary
notscary
ghost
notscary
boo!
notscary
boo!
notscary
ghost
boo!
notscary
ghost
boo!
ghost
notscary
notscary
ghost
ghost
boo!
notscary
ghost
ghost
ghost
boo!
boo!
boo!
boo!
ghost
ghost
boo!
notscary
boo!
ghost
notscary
notscary
notscary
ghost
notscary
ghost
boo!
ghost
ghost
ghost
ghost
ghost
notscary
ghost
boo!
notscary
notscary
notscary
notscary
notscary
notscary
notscary
ghost
notscary
notscary
notscary
ghost
boo!
notscary
boo!
notscary
notscary
ghost
boo!
ghost
ghost
notscary
notscary
boo!
ghost
ghost
ghost
ghost
notscary
boo!
notscary
notscary
ghost
boo!
ghost
notscary
notscary
boo!
ghost
notscary
boo!
boo!
boo!
ghost
boo!
ghost
ghost
boo!
notscary
ghost
boo!
notscary
notscary
ghost
ghost
ghost
boo!
notscary
boo!
boo!
boo!
ghost
boo!
notscary
ghost
ghost
boo!
boo!
ghost
notscary
ghost
boo!
ghost
boo!
notscary
notscary
boo!
notscary
boo!
notscary
notscary
notscary
ghost
notscary
boo!
notscary
ghost
notscary
notscary
notscary
ghost
notscary
ghost
boo!
notscary
notscary
boo!
notscary
boo!
boo!
ghost
notscary
notscary
boo!
boo!
boo!
boo!
boo!
ghost
boo!
ghost
boo!
notscary
boo!
notscary
ghost
ghost
boo!
notscary
ghost
ghost
notscary
notscary
ghost
notscary
notscary
ghost
boo!
boo!
boo!
ghost
ghost
boo!
boo!
boo!
ghost
ghost
boo!
boo!
boo!
ghost
ghost
ghost
notscary
notscary
boo!
boo!
notscary
boo!
ghost
ghost
boo!
ghost
notscary
notscary
boo!
ghost
ghost
ghost
boo!
notscary
boo!
ghost
ghost
boo!
ghost
ghost
ghost
ghost
ghost
boo!
ghost
ghost
ghost
ghost
ghost
ghost
ghost
notscary
boo!
boo!
ghost
ghost
boo!
notscary
ghost
notscary
boo!
boo!
ghost
notscary
ghost
boo!
notscary
ghost
notscary
boo!
boo!
notscary
ghost
boo!
boo!
notscary
boo!
ghost
boo!
boo!
ghost
ghost
ghost
notscary
ghost
boo!
boo!
ghost
boo!
boo!
boo!
boo!
ghost
notscary
notscary
boo!
notscary
ghost
ghost
boo!
boo!
ghost
boo!
notscary
ghost
notscary
notscary
ghost
boo!
boo!
boo!
boo!
ghost
notscary
ghost
notscary
ghost
boo!
ghost
ghost
boo!
boo!
ghost
ghost
boo!
notscary
boo!
boo!
boo!
notscary
ghost
boo!
boo!
ghost
boo!
notscary
ghost
boo!
ghost
notscary
boo!
ghost
ghost
notscary
ghost
boo!
boo!
boo!
ghost
notscary
boo!
notscary
ghost
boo!
boo!
notscary
notscary
ghost
boo!
notscary
ghost
boo!
boo!
notscary
boo!
notscary
boo!
notscary
boo!
ghost
ghost
boo!
notscary
boo!
ghost
ghost
ghost
notscary
ghost
notscary
notscary
ghost
boo!
boo!
boo!
ghost
ghost
boo!
ghost
boo!
notscary
notscary
ghost
notscary
ghost
notscary
notscary
notscary
ghost
ghost
boo!
boo!
notscary
notscary
ghost
ghost
boo!
ghost
boo!
ghost
ghost
boo!
notscary
boo!
ghost
ghost
notscary
notscary
ghost
notscary
ghost
notscary
notscary
boo!
ghost
boo!
notscary
boo!
ghost
notscary
ghost
boo!
ghost
ghost
boo!
ghost
ghost
boo!
notscary
notscary
notscary
ghost
ghost
boo!
notscary
boo!
notscary
ghost
ghost
boo!
ghost
ghost
notscary
notscary
notscary
ghost
notscary
ghost
boo!
ghost
notscary
boo!
ghost
ghost
ghost
ghost
boo!
ghost
notscary
notscary
boo!
boo!
notscary
boo!
ghost
ghost
boo!
notscary
notscary