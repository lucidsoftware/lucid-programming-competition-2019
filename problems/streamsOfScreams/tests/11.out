Ah!
Ah!
Ah!
Ah!
Ah!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ah!
Ahhh!!!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ah!
Ahhh!!!
Ah!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ah!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ah!
Ahhh!!!
Ah!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ah!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ah!
Ah!
Ah!
Ah!
Ah!
Ah!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ah!
Ah!
Ah!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ah!
Ah!
Ahhh!!!
Ah!
Ah!
Ah!
Ah!
Ah!
Ah!
Ah!
Ah!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ah!
Ah!
Ah!
Ahhh!!!
Ah!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ah!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ah!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ahhh!!!
Ah!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ah!
Ah!
Ah!
Ahhh!!!
Ah!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ah!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ah!
Ah!
Ah!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ahhh!!!
Ah!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ah!
Ah!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ah!
Ahhh!!!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ah!
Ahhh!!!
Ah!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ah!
Ahhh!!!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ah!
Ahhh!!!
Ah!
Ahhh!!!
Ah!
Ah!
Ah!
Ah!
Ahhh!!!
Ah!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ah!
Ahhh!!!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ah!
Ahhh!!!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ah!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ahhh!!!
Ah!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ah!
Ahhh!!!
Ah!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ah!
Ahhh!!!
Ah!
Ah!
Ah!
Ahhh!!!
Ah!
Ah!
Ah!
Ah!
Ahhh!!!
Ah!
Ah!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ahhh!!!
Ah!
Ahhh!!!
Ah!
Ah!
Ah!
Ahhh!!!
Ah!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ah!
Ahhh!!!
Ah!
Ah!
Ah!
Ah!
Ah!
Ah!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ah!
Ah!
Ah!
Ah!
Ahhh!!!
Ah!
Ahhh!!!
Ah!
Ahhh!!!
Ah!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ah!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ah!
Ahhh!!!
Ah!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ah!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ah!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ah!
Ahhh!!!
Ah!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ah!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ah!
Ahhh!!!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ah!
Ah!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ah!
Ahhh!!!
Ah!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ah!
Ah!
Ah!
Ah!
Ah!
Ahhh!!!
Ah!
Ah!
Ah!
Ah!
Ah!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ah!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ah!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ahhh!!!
Ah!
Ahhh!!!
Ah!
Ahhh!!!
Ah!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ah!
Ahhh!!!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ahhh!!!
Ah!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ah!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ah!
Ahhh!!!
Ah!
Ah!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ah!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ah!
Ah!
Ahhh!!!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
Ah!
Ah!
Ah!
Ah!
Ahhh!!!
Ah!
Ahhh!!!
Ah!
Ah!
Ah!
Ah!
Ahhh!!!
Ah!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ah!
Ah!
Ahhh!!!
1275
